### PR TITLE
Port 'runner3680 5x7'

### PR DESCRIPTION
**Pull request by silverneko**
Port the '5x7' variant of the 'runner3680' board to VIAL keymap.

Tested with `make omkbd/runner3680/5x7:vial` && flash.

### CODE REVIEW

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/config.h [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+#pragma once
+
+//#define USE_MATRIX_I2C
+
+/* Select hand configuration */
+
+#define MASTER_LEFT
+// #define MASTER_RIGHT
+// #define EE_HANDS
+
```
Comment (lesshonor) [inline keyboards/omkbd/runner3680/5x7/keymaps/vial/config.h]: Master left is default, so no need to specify it.
```suggestion
```

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c [hunk 2]
```diff
@@ -0,0 +1,97 @@
...
+#endif
+
+enum layer_number {
+    _QWERTY = 0,
+    _ADJUST
+};
+
+enum custom_keycodes {
+  QWERTY = SAFE_RANGE,
+  ADJUST,
```
Comment (lesshonor) [inline keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c]:
```suggestion
```
Not sure why this is a custom keycode when you can just use `MO(_ADJUST)`?

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+     * |------+------+------+------+------+------+------|      |------+------+------+------+------+------+------|
+     * |  F5  | Ctrl | GUI  | Alt  | EISU |  Del | Space|      | Enter| Bksp | EISU |  ESC |Adjust| Left | Down |
+     * `------------------------------------------------'      `------------------------------------------------'
+     */
+  [_QWERTY] = LAYOUT(
+      KC_F1,   KC_GRV,  KC_1,    KC_2,    KC_3,    KC_4,    KC_5,       KC_6,    KC_7,    KC_8,    KC_9,    KC_0,    KC_MINS, KC_EQL,
+      KC_F2,   KC_TAB,  KC_Q,    KC_W,    KC_E,    KC_R,    KC_T,       KC_Y,    KC_U,    KC_I,    KC_O,    KC_P,    KC_LBRC, KC_RBRC,
+      KC_F3,   KC_ESC,  KC_A,    KC_S,    KC_D,    KC_F,    KC_G,       KC_H,    KC_J,    KC_K,    KC_L,    KC_SCLN, KC_QUOT, KC_ENT,
+      KC_F4,   KC_LSFT, KC_Z,    KC_X,    KC_C,    KC_V,    KC_B,       KC_N,    KC_M,    KC_COMM, KC_DOT,  KC_SLSH, KC_BSLS, KC_RSFT,
+      KC_F5,   KC_LCTL, KC_LGUI, KC_LALT, EISU,    KC_DEL,  KC_SPC,     KC_ENT,  KC_BSPC, ADJUST,  KC_LEFT, KC_DOWN, KC_UP,   KC_RIGHT
```
Comment (lesshonor) [inline keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c]:
```suggestion
      KC_F5,   KC_LCTL, KC_LGUI, KC_LALT, EISU,    KC_DEL,  KC_SPC,     KC_ENT,  KC_BSPC, MO(_ADJUST),  KC_LEFT, KC_DOWN, KC_UP,   KC_RIGHT
```

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c [hunk 4]
```diff
@@ -0,0 +1,97 @@
...
+        break;
+
+      case ADJUST:
+        if (record->event.pressed) {
+          layer_on(_ADJUST);
+        } else {
+          layer_off(_ADJUST);
+        }
+        break;
+
```
Comment (lesshonor) [inline keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c]:
```suggestion
```

**Review by lesshonor — CHANGES_REQUESTED** [review]

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c [hunk 5]
```diff
@@ -0,0 +1,97 @@
...
+      _______, _______, RGB_MOD, RGB_HUD, RGB_SAD, RGB_VAD, _______,    _______, _______, _______, _______, _______, _______, _______,
+      _______, _______, _______, _______, _______, _______, _______,    _______, _______, _______, _______, _______, _______, _______
+      )
+};
+
+bool process_record_user(uint16_t keycode, keyrecord_t *record) {
+  switch (keycode) {
+      case QWERTY:
+        if (record->event.pressed) {
+           print("mode just switched to qwerty and this is a huge string\n");
```
Comment (lesshonor) [inline keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c]:
```suggestion
```
This print doesn't seem necessary.
Honestly, the whole `QWERTY` custom keycode doesn't seem necessary given there aren't any alternative base layers...

**Review by lesshonor — CHANGES_REQUESTED** [review]

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/config.h (hunk 1 above)
Comment (silverneko) [reply keyboards/omkbd/runner3680/5x7/keymaps/vial/config.h]: Done

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c (hunk 2 above)
Comment (silverneko) [reply keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c]: Suggestion applied.

For the record I'm not the author of the original keymap. I just copied the 'default' keymap and added the vial.json for VIAL to recognize this board. Trying to port this board with minimal modifications.

Realistically speaking, I don't think any people should be using this default keymap unless they need RGB support.
In fact my own custom keymap doesn't need RGB and just use the JSON exported from the QMK configurator web app https://github.com/vial-kb/vial-qmk/commit/7f42dcef63158f274ca490a647a152b59bb3a22c#diff-c92b95e0f107a2aeeb7945459defa50747da6aa4e10653b465e9286eeec29cea

File: keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c (hunk 5 above)
Comment (silverneko) [reply keyboards/omkbd/runner3680/5x7/keymaps/vial/keymap.c]: Agree. Removed.

**Review by lesshonor — APPROVED** [review]

**Pull request merged**